import java.io.IOException;
public class ex77 {
public static void main(String[] args) {
		/*
		try {
		System.out.println("hello");
		} catch (ArithmeticException e) {
		e.printStackTrace();
		}*/
	/*
	 try {
	 System.out.println("hello");
       } catch (Exception e) {
	 System.out.println(e);
	}*/
     /*  try {
         System.out.println("hello");
     }   catch (IOException e) {
      // TODO: handle exception
	 System.out.println(e);
	}*/
		/*
		try {
		System.out.println("hello");
		} catch (InterruptedException e) {
	     // TODO: handle exception
		System.out.println(e);
		}*/
		
		
		/*try {
		System.out.println("hello");
	      } catch (Error e) {
	     // TODO: handle exception
		System.out.println(e);
		}*/
	}
}
